Datagrid/Pager.php

```diff
@@ -247,11 +247,27 @@ public function getPrevious()
     }
 
     /**
+     * NEXT_MAJOR: remove this method
+     *
+     * @deprecated
```

Comment:
Please add a message to the `@deprecated` annoation like we do for any other method